src/init/system_start_service.c

```diff
@@ -22,12 +25,20 @@ static const char *script_commands[] = {
 
 int system_start(void) {
 	const char *command;
+	char *argv[10];
+	int argc;
+	const struct cmd *cmd;
 
 	setup_tty(OPTION_STRING_GET(tty_dev));
 
 	array_foreach(command, script_commands, ARRAY_SIZE(script_commands)) {
-
-		system(command);
+		argc = cmdline_tokenize((char *)command, argv);
+		if (0 == strncmp(argv[0], "pthread", 7)) {
```

Comment:
May be, but I want to make them in static, for resource restrictions system.
For example robots in previous init mode (start_sctipt) we must set command as build_in in tish. Of cause tish must be enable and when it stating tish use the same stack as  running cmd (the only existing kernel stack). Now cmd can be call directly, of cause it save just a couple of bytes but in small systems it may be impotent/